Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,232 @@
+
+Users can also publish on their local Read-write-web play. In order to do this, a user must run the following commands inside the local rww-play directory.
+
+```bash
+$ git 
+> publishLocal
+```
+
+One can then import it into their scripts from inside the ammonite shell. The user should then be able to make a get on their local rww-play and check whether they can access a resource that has access control. This means that the user will need to make themselves a key.
+
+## The Key Pair
+
+A key pair is a pair formed from a Public and a Private key ----------- (more info on public and private keys ?)
+
+### Resolving dependencies
+
+In order to create this pair, the user must first import several dependencies from inside their ammonite shell.
+
+```scala
+@ import coursier.core.Authentication, coursier.MavenRepository
+
+@ interp.repositories() ++= Seq(MavenRepository(
+     "http://bblfish.net/work/repo/snapshots/"
+  ))
+
+@ import $ivy.`org.w3::banana-jena:0.8.5-SNAPSHOT`
+@ import org.w3.banana._
+@ import org.w3.banana.syntax._
+@ import org.w3.banana.jena.Jena
+@ import Jena._
+@ import Jena.ops._
+```
+
+This will make sure that all of the Ivy dependencies are correctly resolved, getting information from the bblfish.net repository. After that the user must import the AkkaHttp signature support and the key functionality.
+
+```scala
+@ import $ivy.`run.cosy::akka-http-signature:0.2-SNAPSHOT`
+@ import run.cosy.auth.RSAKeys
+``` 
+After that, the user should be able to make a key pair using the imported libraries.
+
+### Creating and using a key pair
+
+A user can create a key pair using the commands found within the [Akka-http-signature Source code](https://github.com/fstoqnov/akka-http-signature/tree/master/src).
+
+```scala
+@ val (pub,priv) = RSAKeys.buildRSAKeyPair()
+
+pub: java.security.interfaces.RSAPublicKey = Sun RSA public key, 2048 bits
+  modulus: 18617616591683306896114159596818930151392321021655743092554571476906272216221226120621265951005438129779697999635487320523329319543618461573656972895570742121788481588357965539288547158041270238222755734178260848243117538931191184092136701393910768801017892322044665470799614169981926296276068464740129396666480935344897491410772746293264767223181696074228079861695236831569819678997680346803798366156295040872828579274684209593397084757356046266872087618604860475724560124477904326733099073845912710038077198861072318221134249054814455997792289530305256172821638522445655947916737229103835394051258962680413560258903
+  public exponent: 65537
+priv: java.security.interfaces.RSAPrivateKey = sun.security.rsa.RSAPrivateCrtKeyImpl@ffd26868
+```
+
+This command will generate a Public and Private key pair. As we can see only information about the Public key is being displayed in BigInt notation. The private key will be hidden from view - only the reference memory location will be shown.
```

Comment:
It is not "BigInt" notation, it is "decimal notation". BigInt is the object that allows one in Java to have indefinitely long numbers.